cds_ils/migrator/default_records.py

```diff
@@ -15,12 +15,15 @@
 from invenio_app_ils.ill.api import LIBRARY_PID_TYPE
 from invenio_app_ils.ill.proxies import current_ils_ill
 from invenio_app_ils.proxies import current_app_ils
+from invenio_app_ils.series.api import SERIES_PID_TYPE
 from invenio_db import db
 from invenio_pidstore.models import PersistentIdentifier, PIDStatus
 
 MIGRATION_DOCUMENT_PID = "DOCPID-44444"
 MIGRATION_LIBRARY_PID = "LIBPID-44444"
 MIGRATION_VENDOR_PID = "VENPID-44444"
+MIGRATION_YELLOW_PID = "YELLO-44444"
```

Comment:
I see that the values of `doc/lib/vendor` are 6 letters - 5 digits, while we always use 5 - 5. Does this create any issue? ping @kprzerwa 
I would probably go for 5 - 5

Reply:
I tested it and was working correctly, but I can change it to be sure.